src/dependencies.jl

```diff
@@ -864,15 +864,15 @@ macro install(_libmaps...)
                         depsfile_buffer = IOBuffer()
                         println(depsfile_buffer,
                             """
-                            # This is an auto-generated file; do not edit
+                            # This is an auto-generated file; do not edit this file.
```

Comment:
changing the content here (and below) will force every package to rebuild unnecessarily